core/src/tile/tileManager.cpp

```diff
@@ -237,6 +238,9 @@ void TileManager::updateTileSet(TileSet& _tileSet, const ViewState& _view,
                 entry.task = _tileSet.source->createTask(visTileId);
                 enqueueTask(_tileSet, visTileId, _view);
                 m_tilesInProgress++;
+            } else if (!entry.needsLoading() && !entry.isReady()) {
+                // tile not generated yet!
+                m_tilesInGeneration++;
             }
```

Comment:
Do we need to differentiate `m_tilesInProgress` and `m_tilesInGeneration`? Otherwise I would count this case as 'in progress'